cluster/gce/trusty/configure-helper.sh

```diff
@@ -147,9 +147,7 @@ assemble_kubelet_flags() {
   if [ -n "${KUBELET_PORT:-}" ]; then
     KUBELET_CMD_FLAGS="${KUBELET_CMD_FLAGS} --port=${KUBELET_PORT}"
   fi
-  if [ -n "${KUBELET_TEST_ARGS:-}" ]; then
-    KUBELET_CMD_FLAGS="${KUBELET_CMD_FLAGS} ${KUBELET_TEST_ARGS}"
-  fi
+  KUBELET_CMD_FLAGS="${KUBELET_CMD_FLAGS} ${KUBELET_TEST_ARGS:-}"
```

Comment:
The order does not matter. The var ${KUBELET_TEST_ARGS} only provides additional flags for tests, but does not cover all flags assembled above. It used to have "--v=4", but a couple of months ago, the log level was split in ${KUBELET_TEST_LOG_LEVEL}. If we write the line 143 as KUBELET_CMD_FLAGS="--v=2 ${KUBELET_TEST_ARGS:-}", then the lines 144--146 will make it "--v=2 --v=4"